websockets/README.md

```diff
@@ -0,0 +1,69 @@
+# libp2p WebSockets
+
+| Lifecycle Stage | Maturity                 | Status | Latest Revision |
+|-----------------|--------------------------|--------|-----------------|
+| 2A              | Candidate Recommendation | Active | r0, 2022-10-12  |
```

Comment:
```suggestion
| 3A              | Recommendation | Active | r0, 2022-10-12  |
```

Should this be a Recommendation instead of candidate? Given that more than there more than 2 reference implementations of this spec?

Reply:
Sounds good, have merged the suggestion